ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -152,7 +162,10 @@ def initialize_trainers(self, trainer_config):
                         .brains_to_curriculums[brain_name]
                         .min_lesson_length if self.meta_curriculum else 0,
                     trainer_parameters_dict[brain_name],
-                    self.train_model, self.load_model, self.seed, self.run_id)
+                    self.train_model, self.load_model, self.seed,
+                    self.run_id)
+                if hasattr(self.trainers[brain_name], 'trainer_metrics'):
```

Comment:
Don't they all have the attribute sine trainer_metrics is declared in the abstract trainer?

Reply:
Yep, this was an artifact of our older design where the `trainer_metric` was part of just `PPOTrainer`.  I will fix.